README.md

```diff
@@ -54,12 +61,12 @@ also the topic where these events are produced. Make sure that you can read them
 
 - Single event:
 ```
-curl -X POST http://localhost:5000/extractor_sql/mimic-admissions/10013
+curl -X POST http://localhost:5000/extractor_sql/admissions/10013
```

Comment:
does `admissions` work at the moment or only `patients` ?

Reply:
It's a bit tricky, the resource-id (admission, patient, ...) provided in the route isn't used, just passed as is in the event

Reply:
I'll change the route a bit